routing/result_interpretation_test.go

```diff
@@ -96,13 +96,17 @@ var (
 				AmtToForward: 99,
 			},
 			{
-				PubKeyBytes:   hops[2],
-				AmtToForward:  95,
+				PubKeyBytes: hops[2],
+				// The amount for the intermediate blinded hop
```

Comment:
nit: it could be clearer if you write `Intermediate blinded hops don't have an amount set.`, otherwise it may be misleading that you describe your action of setting the amount to zero here.